enterprise/app/models/enterprise/channelable.rb

```diff
@@ -0,0 +1,31 @@
+module Enterprise::Channelable
+  extend ActiveSupport::Concern
+
+  # Active support concern has `included` which changes the order of the method lookup chain
+  # https://stackoverflow.com/q/40061982/3824876
+  # manually prepend the instance methods to combat this
+  included do
+    prepend InstanceMethods
+  end
+
+  module InstanceMethods
+    def create_audit_log_entry
+      account = self.account
+      associated_type = 'Account'
+      auditable_id = inbox.id
+      auditable_type = 'Inbox'
+      audited_changes = saved_changes.except('updated_at')
+
+      return if audited_changes.blank?
```

Comment:
Could also add a condition return if inbox.nil? 

for the cases where we don't want this to be triggered , ie inbox delete

Reply:
done